src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -51,28 +51,52 @@ module SoilBiogeochemDecompCascadeBGCMod
   integer, private, parameter :: i_litr1   = i_met_lit  ! First litter pool, metobolic
   integer, private, parameter :: i_litr2   = i_cel_lit  ! Second litter pool, cellulose
   integer, private, parameter :: i_litr3   = i_lig_lit  ! Third litter pool, lignin
+  integer, private, parameter :: i_micr1   = i_micr_r   ! First microbial pool, copiotrophic, as labeled in Wieder et al. 2015
+  integer, private, parameter :: i_micr2   = i_micr_k   ! Second microbial pool, oligotrophic, as labeled in Wieder et al. 2015
 
   type, private :: params_type
      real(r8):: cn_s1_bgc     !C:N for SOM 1
      real(r8):: cn_s2_bgc     !C:N for SOM 2
      real(r8):: cn_s3_bgc     !C:N for SOM 3
 
-     real(r8):: rf_l1s1_bgc   !respiration fraction litter 1 -> SOM 1
-     real(r8):: rf_l2s1_bgc
-     real(r8):: rf_l3s2_bgc
+     ! TODO slevis: New rf params in MIMICS. Update params file accordingly.
+     !              I haven't identified what these correspond to in testbed.
+     real(r8):: rf_l1m1_mimics  !respiration fraction litter 1 -> microbial 1
+     real(r8):: rf_l2m1_mimics
+     real(r8):: rf_l3m1_mimics
 
-     real(r8):: rf_s2s1_bgc    
-     real(r8):: rf_s2s3_bgc    
-     real(r8):: rf_s3s1_bgc    
+     real(r8):: rf_l1m2_mimics
+     real(r8):: rf_l2m2_mimics
+     real(r8):: rf_l3m2_mimics
 
+     real(r8):: rf_s1s3_mimics  ! respiration fraction soil 1 -> soil 3
+     real(r8):: rf_s2s3_mimics
+     real(r8):: rf_s3m1_mimics
+     real(r8):: rf_s3m2_mimics
+
+     real(r8):: rf_m1s1_mimics
+     real(r8):: rf_m1s2_mimics
+     real(r8):: rf_m1s3_mimics
+
+     real(r8):: rf_m2s1_mimics
+     real(r8):: rf_m2s2_mimics
+     real(r8):: rf_m2s3_mimics
+
+     ! TODO slevis: Keep these?
      real(r8):: rf_cwdl2_bgc 
      real(r8):: rf_cwdl3_bgc
 
-     real(r8):: tau_l1_bgc    ! 1/turnover time of  litter 1 from Century (l/18.5) (1/yr)
-     real(r8):: tau_l2_l3_bgc ! 1/turnover time of  litter 2 and litter 3 from Century (1/4.9) (1/yr)
-     real(r8):: tau_s1_bgc    ! 1/turnover time of  SOM 1 from Century (1/7.3) (1/yr)
-     real(r8):: tau_s2_bgc    ! 1/turnover time of  SOM 2 from Century (1/0.2) (1/yr)
-     real(r8):: tau_s3_bgc    ! 1/turnover time of  SOM 3 from Century (1/0.0045) (1/yr)
+     ! TODO slevis: tau params are functions (Vmax and Km) in MIMICS. Also new
+     !              tau params define microbial turnover to SOM. Update params
+     !              file if necessary.
+     real(r8):: tau_l1_mimics     ! 1/turnover time of litter 1 (1/yr)
+     real(r8):: tau_l2_l3_mimics  ! 1/turnover time of litter 2 and 3 (1/yr)
+     real(r8):: tau_s1_mimics     ! 1/turnover time of SOM 1 (1/yr)
+     real(r8):: tau_s2_mimics     ! 1/turnover time of SOM 2 (1/yr)
+     real(r8):: tau_s3_mimics     ! 1/turnover time of SOM 3 (1/yr)
+     real(r8):: tau_m1_mimics     ! 1/turnover time of microbial 1 (1/yr)
+     real(r8):: tau_m2_mimics     ! 1/turnover time of microbial 2 (1/yr)
+     ! TODO slevis: Keep?
```

Comment:
here I think we just want the turnover of microbial pools `tau_m1_mimics` & `tau_m2_mimics`, which are actually functions.  I don't know how best to implement the Vmax and Km functions that are needed to calculate decompositon rates in MIMICS.  Maybe:
`Vmax_l1_m1_mimics`, `Vmax_l2_m1_mimics`,`Vmax_s2_m1_mimics` etc.

Reply:
below, we'll use tau_cwd_bgc, but cwd_fcel_bgc is not relevent.